standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/repl/DumpDirCleanerTask.java

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
This should remain part of hive. repl dump command is implemented in hive and the cleanup of that also should be in a thread whose impl is in hive.
I guess this falls under same bucket as compactor thread. What is the approach being followed for that ?

Reply:
For now the compactor is in Hive because it's tightly entwined in ql, though I plan to move it someday (though Eugene doesn't agree with me on that yet :) ).  The compactor really needs to move because without it you cannot operate on ACID files.

For this, I'm fine with leaving it in Hive.  I've wondered in general if the repl stuff should come along, though parts already have because of its tight integration into HiveMetaStore.

Reply:
Replication can be considered to have 2 parts - metastore event logging (DBNotificationListner) and the repl/dump load.
In my opinion, event logging belongs to metastore. The repl load/dump commands (they are sql commands, that generate query plans) should be in Hive. So associated code, such as cleanup of the repl dump dir etc should also be in Hive.

Reply:
Makes sense.  I can move this class back into metastore.